packages/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -33,16 +33,38 @@ export interface TransactionFunctions {
 	): boolean;
 }
 
-interface TransactionPoolOptions {
+export interface CheckTransactionsResult {
+	transactionsResponses: ReadonlyArray<TransactionResponse>;
+	status: boolean;
+}
+
+interface TransactionResponse {
+	readonly errors: ReadonlyArray<Error>;
+	readonly id: string;
+	readonly status: boolean;
```

Comment:
this will be enum Status.OK / Status.FAIL, so 0 or 1.

Reply:
Is there any particular reason for making it Enum? Because there can't be more than two options.

Reply:
with the name `status` decided in the meeting, `status = true` doesn't make sense semantically.

Reply:
I think Status.OK / FAIL enum is easy to read but if you want I can change it to simply 0 or 1. compile result would be the same.